BookAPI/Controllers/AuthorController.cs

```diff
@@ -59,7 +59,7 @@ public async Task<IActionResult> AddAuthor(AuthorViewModel author)
         // Put Update
         [HttpPut]
         [Route("{name}")]
-        public async Task<IActionResult> Edit([FromRoute] string name, AuthorViewModel updateAuth)
+        public async Task<IActionResult> Edit([FromRoute] string name, [FromBody]AuthorViewModel updateAuth)
```

Comment:
why we're still getting name from Route?

Reply:
because with fromroute it is mandatory; if I add fromquery, it becomes optional

Reply:
this is the author name, right?